src/utils.ts

```diff
@@ -3,6 +3,21 @@ import type { BigNumber } from "ethers";
 import multibase from 'multibase';
 import multihashes from 'multihashes';
 import type { Config } from '@app/config';
+import type { Registration } from "@app/base/registrations/registrar";
+import { getRegistration } from '@app/base/registrations/registrar';
+
+export interface radProfile {
+  name: string;
+  avatar: string;
+  url: string;
+  github: string;
+  twitter: string;
+}
```

Comment:
Probably this is something I'd move to a `profile.ts` file, if that makes sense, and it should just be called `Profile`.

Is eslint okay with non-capitalized type names?

Reply:
Seems like it's okay by default, so I added a lint for it on master, with a couple of other things, so our variable/type names are consistent!

Reply:
This commit 008cd890ab5a0b7c2b668509bec1fbbc098af819 uses BasicProfile as type of `@ceramicstudio/idx-constants`